workload/src/Samsung.Tizen.Build.Tasks/CheckApiVersion.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using Samsung.Tizen;
+
+namespace Samsung.Tizen.Build.Tasks
+{
+    public class CheckApiVersion : Task
+    {
+        [Required]
+        public string ManifestApiVersion { get; set; }
+
+        [Required]
+        public string TargetFrameworkIdentifier { get; set; }
+
+        [Required]
+        public string TargetPlatformIdentifier { get; set; }
+
+        [Required]
+        public string TargetFrameworkVersion { get; set; }
+
+        [Required]
+        public string TargetPlatformVersion { get; set; }
+
+        [Required]
+        public ITaskItem[] SupportedAPILevelList { get; set; }
+
+        Version ApiVersion;
+
+        public override bool Execute()
+        {
+            if (string.IsNullOrEmpty(ManifestApiVersion))
+            {
+                Log.LogError("ManifestApiVersion is Invalid {0}", ManifestApiVersion);
+                return false;
+            }
+
+            if (TargetFrameworkIdentifier == ".NETCoreApp" && TargetPlatformIdentifier == "tizen")
+            {
+                //net6.0 tfm, get api version from tpv
+                if (string.IsNullOrEmpty(TargetPlatformVersion))
+                {
+                    Log.LogError("TargetPlatformVersion is Invalid {0}", TargetPlatformVersion);
+                    return false;
+                }
+                Version.TryParse(TargetPlatformVersion, out ApiVersion);
+            }
+            else
+            {
```

Comment:
Please check whether  TFI is "Tizen" or not. 
Some VD application uses `netcoreapp2.0` TFM.

Reply:
fixed